lib/gds_api/content_data_api.rb

```diff
@@ -19,12 +19,18 @@ def content_data_api_endpoint
     "#{Plek.current.find('content-performance-manager')}/api/v1"
   end
 
-  def query(from:, to:, metrics:)
-    query_params = {}
-    query_params[:from] = from
-    query_params[:to] = to
-    query_params[:metrics] = metrics
+  def content_summary(from:, to:, organisation:)
```

Comment:
Not sure if `content_summary` represents what the endpoint does.
Can you think of a better name? It doesn't feel to me like a summary,
but a list of content items.